### PR TITLE
add deviantArt provider #90

### PR DESCRIPTION
**Pull request by TerribleDev**
Add DeviantArt provider for issue #90

### CODE REVIEW

**Comment by TerribleDev** [comment]
this seems to work fine

**Comment by TerribleDev** [comment]
ok this should be working fine, if anyone wants to check it they can get a key from [DeviantArt](https://www.deviantart.com/developers/)

File: Owin.Security.Providers/DeviantArt/DeviantAuthenticationOptions.cs [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        public ISecureDataFormat<AuthenticationProperties> StateDataFormat { get; set; }
+
+        /// <summary>
+        ///     Initializes a new <see cref="DeviantArtAuthenticationOptions" />
+        /// </summary>
+        public DeviantArtAuthenticationOptions()
+            : base("DeviantArt")
+        {
+            Caption = Constants.DefaultAuthenticationType;
+            CallbackPath = new PathString("/signin-DeviantArt");
```
Comment (jerriep) [inline Owin.Security.Providers/DeviantArt/DeviantAuthenticationOptions.cs]: Once again, for consistency please make the callback path lowercase

**Comment by TerribleDev** [comment]
@jerriep ....done

**Comment by TerribleDev** [comment]
@jerriep  I fixed the merge conflict here

**Comment by TerribleDev** [comment]
@jerriep it seems there were some merging weirdness with the untappd PR. I added a fix to this PR, essentially the untappd files were being declared twice in the csproj

**Comment by jerriep** [comment]
@tparnell8 We are looking for help moving to ASP.NET 5. Please read http://www.jerriepelser.com/blog/calling-contributors-for-oauth-providers

**Pull request merged**